src/test/java/io/sniffy/socket/SnifferInputStreamTest.java

```diff
@@ -0,0 +1,112 @@
+package io.sniffy.socket;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.mockito.Matchers.anyInt;
+import static org.mockito.Mockito.reset;
+import static org.mockito.Mockito.verify;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SnifferInputStreamTest {
+
+    private static final byte[] DATA = new byte[]{1,2,3,4};
+
+    @Mock
+    private SnifferSocketImpl snifferSocket;
+
+    @Test
+    public void testReadByteByByte() throws IOException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://www.codacy.com/app/sniffy/sniffy/file/2492520317/issues/source?bid=3005960&fileBranchId=3034483#l28)